include/libdnf/utils/utils.hpp

```diff
@@ -89,6 +105,54 @@ inline GoalProblem operator&(GoalProblem lhs, GoalProblem rhs) {
         static_cast<std::underlying_type<GoalProblem>::type>(rhs));
 }
 
+inline bool GoalSettings::get_strict(libdnf::ConfigMain & cfg_main) {
+    bool ret;
```

Comment:
Just a note, this temporary variable is unnecessary in this and the below two methods. Results in multiple redundant LOCs... I've fixed it here: https://github.com/rpm-software-management/libdnf/pull/1184/commits/6eb0a9b19fd6cc47c65a4cf1f5056748675284c3